build_tools/benchmarks/convperf/build_and_run_convperf.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Sets up `convperf` (https://github.com/nod-ai/convperf).
+#
+# `convperf` benchmarks convolution workloads on IREE and other backends such
+# as libxsmm. IREE is included as a submodule.
+#
+# Currently x86 CPU is supported.
+#
+# Usage:
+#    ./build_and_run_convperf.sh \
+#        <convperf build dir> \
+#        <convperf results dir> \
+#        <iree SHA> \
+#        <convperf repo dir> (optional)
+
+set -xeuo pipefail
+
+export BUILD_DIR=$1
+export RESULTS_DIR=$2
+export IREE_SHA=${3:-"origin/main"}
+export REPO_DIR=${4:-${CONVPERF_REPO_DIR}}
+
+pushd ${REPO_DIR}
+source convperf.venv/bin/activate
+
+# Set all repos as a safe directory. Since this repo was created in the
+# DockerFile under `root`, git will not run commands on this repo as a
+# non-root user unless it is marked safe.
```

Comment:
This can be separate, but I'm curious whether this is something that could be done when building the container itself. Less setup to use it would be nice.

Reply:
I tried this for mmperf. When moving it to the container, I get the unsafe directory error once the `build_and_run_convperf.sh` runs. The directories are added to the .gitconfig so I think this is user dependent. If we run it in the container, it'll be added to the root user's .gitconfig.

Reply:
You should be able to set this with `--system` instead of `--global` to set the overall git system config. See https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration